mindsdb/integrations/handlers/pypi_handler/pypi_handler.py

```diff
@@ -0,0 +1,30 @@
+import logging
+
+import requests
+
+from mindsdb.integrations.handlers.pypi_handler.pypi_tables import PyPIRecentTable
+from mindsdb.integrations.libs.api_handler import APIHandler
+from mindsdb.integrations.libs.response import HandlerStatusResponse as StatusResponse
+from mindsdb.utilities.log import get_log
+
+logger = get_log("integrations.pypi_handler")
+
+
+class PyPIHandler(APIHandler):
+    def __init__(self, name: str, **kwargs) -> None:
+        super().__init__(name)
+
+        connection_data = kwargs.get("connection_data", {})
+        self.package = connection_data.get("package", None)
+
+        pypi_recent_data = PyPIRecentTable(self)
+        self._register_table("recent", pypi_recent_data)
+
+    def check_connection(self):
+        return StatusResponse(True)
+
+    def connect(self):
```

Comment:
Hey @lnxpy,
I thought I would reply here since you asked. Since you don't seem to need to instantiate an object or anything here, I believe you can just return the module itself in your `connect()` method.
```
def connect(self):
        return pypistats
```

You can now use this within your `Table` classes similar to the following example:
```
connection = self.handler.connect()
df = connection.recent('<some-package>', total=True, format="pandas")
```

Reply:
Hi @MinuraPunchihewa,
Thanks.